ddtrace/contrib/aiohttp/__init__.py

```diff
@@ -1,4 +1,6 @@
 """
+Note: This integration requires additional setup for automatic instrumentation to be applied.
+
 The ``aiohttp`` integration traces all requests defined in the application handlers.
 Auto instrumentation is available using the ``trace_app`` function::
```

Comment:
I wonder if instead we should adjust the "Auto instrumentation is available".

```suggestion
The ``aiohttp`` integration traces all requests defined in the application handlers.

Automatic instrumentation is not available for ``aiohttp.web.Application``, the provided ``trace_app`` function must be used::
```


?